internal/it/util.go

```diff
@@ -469,6 +474,45 @@ func xmlSSLConfig(clusterName string, port int) string {
 			`, clusterName, port)
 }
 
+func xmlCPConfig(clusterName string, port int) string {
+	return fmt.Sprintf(`
+        <hazelcast xmlns="http://www.hazelcast.com/schema/config"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://www.hazelcast.com/schema/config
+            http://www.hazelcast.com/schema/config/hazelcast-config-4.0.xsd">
+            <cluster-name>%s</cluster-name>
+            <network>
+               <port>%d</port>
+            </network>
+			<map name="test-map">
+				<map-store enabled="true">
+					<class-name>com.hazelcast.client.test.SampleMapStore</class-name>
+				</map-store>
+			</map>
+			<map name="test-map-smart">
+				<map-store enabled="true">
+					<class-name>com.hazelcast.client.test.SampleMapStore</class-name>
+				</map-store>
+			</map>
+			<map name="test-map-unisocket">
+				<map-store enabled="true">
+					<class-name>com.hazelcast.client.test.SampleMapStore</class-name>
+				</map-store>
+			</map>
```

Comment:
Are these lines necessary for CP subsystem tests?

Reply:
Actually no, removed at https://github.com/hazelcast/hazelcast-go-client/pull/901/commits/628c21c86995619aae201315bf28b9aef639102a.